bundles/azure-service-broker-0.0.1/chart/azure-service-broker/templates/post-install-job.yaml

```diff
@@ -59,3 +59,58 @@ spec:
                         echo "ServiesBroker is not ready. Timeout reached";
                         exit 1;
                       fi
+---
```

Comment:
I'm not convinced that adding that logic to the existing files is a proper approach. 

IMO we should just have a separate directory for all that stuff: docsTopics, checker Job, ServiceAccounts, RoleBinding etc. 

It will be much easier to distinguish what we added and what comes from upstream :)